### PR TITLE
Fallback to index.html in serveBinaris

### PR DESCRIPTION
**Pull request by bergundy**

### CODE REVIEW

File: server-function/src/serveBinaris.ts [hunk 1]
```diff
@@ -58,6 +58,7 @@ app.post('/invoke', express.json(), async (req, res) => {
 });
 
 app.use(express.static(buildDir));
+app.use((_req, res) => res.sendfile(pathResolve(buildDir, 'index.html')));
```
Comment (vogre) [inline server-function/src/serveBinaris.ts]: Can you make sure content-type / charset match the ones served by static serving?

File: app-testsuite/cypress/integration/full_spec.js [hunk 2]
```diff
@@ -28,8 +28,7 @@ describe('hack', () => {
 
 describe('user HTTP handler', () => {
   it('gets from express handler', () => {
-    cy.visit('/');
+    cy.visit('/express');
```
Comment (vogre) [inline app-testsuite/cypress/integration/full_spec.js]: Cool, now it'll be tested.

File: server-function/src/serveBinaris.ts (hunk 1 above)
Comment (vogre) [reply server-function/src/serveBinaris.ts]: Express res.sendfile seems deprecated and recommends res.sendFile instead

Internally it's implemented yhe same as static so content-type should be correct

Comment (bergundy) [reply server-function/src/serveBinaris.ts]: `Content-Type: text/html; charset=UTF-8`, it's the same.
Changing to `sendFile` and on `GET` requests.

**Pull request merged**